client/widgets/db-stats/stats.directive.js

```diff
@@ -0,0 +1,80 @@
+angular.module('probedock.dbStatsWidget').directive('dbStatsWidget', function() {
+  return {
+    restrict: 'E',
+    controller: 'DbStatsWidgetCtrl',
+    templateUrl: '/templates/widgets/db-stats/stats.template.html',
+    scope: {
+      organization: '='
+    }
+  };
+}).controller('DbStatsWidgetCtrl', function(api, $scope) {
+  _.defaults($scope, {
+    loading: true
+  });
+
+  fetchStats();
+
+  function fetchStats() {
+    $scope.loading = true;
+
+    return api({
+      url: '/platformManagement/dbStats'
+    }).then(function(response) {
+      if (response.data) {
```

Comment:
Superfluous `if`: `response.data` will always pass this condition even if empty.